app/src/main/java/com/azuresamples/msalnativeauthandroidkotlinsampleapp/AccessApiFragment.kt

```diff
@@ -144,26 +153,38 @@ class AccessApiFragment : Fragment() {
         }
     }
 
-    private fun accessWebAPI() {
+    private suspend fun getAccessToken(accountState: AccountState, scopes: List<String>): String {
+        val accessTokenState = accountState.getAccessToken(false, scopes)
+        return if (accessTokenState is GetAccessTokenResult.Complete) {
+            accessTokenState.resultValue.accessToken
+        } else {
+            throw Exception("Failed to get access token")
+        }
+    }
+
+    private suspend fun useAccessToken(WEB_API_BASE_URL: String, accessToken: String): Response {
```

Comment:
nit: rename `WEB_API_BASE_URL` to `url`. it's not a base URL, and it shouldn't be capitalised because it's not a constant.